frontend_lib/src/component/PopinFixed/PopinFixed.styl

```diff
@@ -119,6 +119,7 @@
       &__top
         display flex
         align-items center
+        min-height 38px
```

Comment:
Maybe add a little explanation of why this number ?

Reply:
fixed